bindings/python/pinocchio/shortcuts.py

```diff
@@ -183,6 +297,8 @@ def buildModelsFromMJCF(
         )
 
     lst = [model]
+    if contacts:
+        lst.append(contact_models)
```

Comment:
contact can be set to true but contact_models can be uninitialized.

Before calling buildModelFromMJCF (line 279) create contact_models as an empty list.